extensions/rich_text_components/rte-output-display.component.ts

```diff
@@ -51,6 +51,7 @@ export class RteOutputDisplayComponent implements AfterViewInit {
   @ViewChild('tabs') tabsTagPortal: TemplateRef<unknown>;
   @ViewChild('video') videoTagPortal: TemplateRef<unknown>;
   @Input() rteString: string;
+  @Input() showAltText: boolean = false;
```

Comment:
Here and elsewhere, avoid starting a variable name with a verb. See [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide#javascript) for suggestions.

Reply:
Done, PTAL.